src/services/webclient.ts

```diff
@@ -2410,7 +2410,7 @@ export class WebClientService {
             device: data.device,
             os: data.os,
             osVersion: data.osVersion,
-            isWork: data.isWork,
+            isWork: hasValue(data.isWork) ? data.isWork : false,  // Backwards compat hack, remove after August 2019
```

Comment:
This is fixed in master, but let's leave the fix in here for a while.

Reply:
Nit: I would add this as a `TODO`. Easier to track.